checker-qual/src/main/java/org/checkerframework/checker/initialization/qual/PolyInitialized.java

```diff
@@ -0,0 +1,26 @@
+package org.checkerframework.checker.initialization.qual;
+
+import org.checkerframework.framework.qual.PolymorphicQualifier;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A polymorphic qualifier for the freedom-before-commitment initialization tracking type-system.
+ *
+ * <p>Any method written using {@link PolyInitialized} conceptually has two versions: one in which
+ * every instance of {@link PolyInitialized} has been replaced by {@link Initialized}, and one in
+ * which every instance of {@link PolyInitialized} has been replaced by {@link
```

Comment:
What about the third FbC qualifier, `UnderInitialization`?

Reply:
Agreed. Already added!